### ISSUE TITLE
include package bookmark and add \bookmarksetup{startatroot} to \backmatter

### ISSUE THREAD

**Issue by timmli** [langsci/latex#22]

**Comment by Glottotopia** [comment]
is this still open?

**Comment by Glottotopia** [comment]
bump

**Comment by timmli** [comment]
Yes, it still open. This is the output with the latest test file for monographs (see 17a320d78035e2f7a5c3debf13bb1c4f663badec):

![unbenannt](https://user-images.githubusercontent.com/4593031/45906231-a05dbb00-bdf3-11e8-9917-c17526103879.PNG)

"I Part test" should be on the same level as "References" and "Index".
